src/Uno.UI/MixinGeneration.targets

```diff
@@ -1,38 +1,32 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <Project>
 
-  <PropertyGroup>
-	<IsMonoAndroid>false</IsMonoAndroid>
-	<IsMonoAndroid Condition="
+	<PropertyGroup>
+		<IsAndroid>false</IsAndroid>
+		<IsAndroid Condition="
 				   '$(TargetFramework)'=='net6.0-android'
 				   or '$(TargetFramework)'=='net7.0-android'
 				   or $(TargetFramework.ToLower().StartsWith('monoandroid'))
-				   ">true</IsMonoAndroid>
+				   ">true</IsAndroid>
 
-	<IsXamarinIOS>false</IsXamarinIOS>
-	<IsXamarinIOS Condition="
+		<IsIOS>false</IsIOS>
+		<IsIOS Condition="
 				  '$(TargetFramework)'=='net6.0-ios'
 				  or '$(TargetFramework)'=='net6.0-maccatalyst'
 				  or '$(TargetFramework)'=='net7.0-ios'
 				  or '$(TargetFramework)'=='net7.0-maccatalyst'
 				  or $(TargetFramework.ToLower().StartsWith('xamarinios'))
-				  ">true</IsXamarinIOS>
+				  ">true</IsIOS>
 
-	<IsXamarinMac>false</IsXamarinMac>
-	<IsXamarinMac Condition="
+		<IsMac>false</IsMac>
+		<IsMac Condition="
 				  $(TargetFramework.ToLower().StartsWith('xamarinmac'))
 				  or '$(TargetFramework)'=='net6.0-macos'
 				  or '$(TargetFramework)'=='net7.0-macos'
-				  ">true</IsXamarinMac>
-	
-	<IsReferenceAPI>false</IsReferenceAPI>
-	<IsReferenceAPI Condition="'$(TargetFramework)'=='netstandard2.0' or '$(TargetFramework)'=='net5.0'">true</IsReferenceAPI>
+				  ">true</IsMac>
 
-	<!-- Required because links don't work in actual folder structure -->
-	<_IsUnoUIProjectForMixins Condition="'$(MSBuildProjectName)'=='Uno.UI'">true</_IsUnoUIProjectForMixins>
```

Comment:
This property doesn't seem to be read anywhere. So I deleted it.

Reply:
It was used below, be it's likely that this was a workaround for an older version of VS. It's most likely not useful anymore because many other projects use t4 files and are not named "uno.ui", you can remove the choose block.

Reply:
Fixed.